### DISCUSSION TITLE
TakeLast operator does not yield any items.

### DISCUSSION THREAD

**Discussion by azeebazy** [reactivemarbles/DynamicData#682] [General]
Hi, I am using DynamicData packagelatest version 7.12.11

`
mySourceList.Connect().WhereReasonsAre(ListChangeReason.Add).Do(msg =>
                        {
                            Console.WriteLine("beforeTakeLast");
                        }).TakeLast(1000).Do(msg =>
                        {
                            Console.WriteLine("AfterTakeLast");
                        }).Subscribe();
`

beforeTakeLast is printing properly for all the changes in sourceList.
But the changes does not propogate after TakeLast and line "AfterTakeLast" is never printed.
Am I missing something.

**Comment by RolandPheasant** [comment]
Yes, although DD is based on Rx, it provides its own set of operators. TakeLast is not one of them
